client/src/pages/FlightData/tabs/FlightPlan/FlightPlanToolbar.js

```diff
@@ -150,6 +275,91 @@ const FlightPlanToolbar = props => {
 						<span>ft</span>
 					</div>
 				</Row>
+				<span>Place Point:</span>
+				<Row columns="minmax(0, 2fr) minmax(0, 2fr) minmax(0, 2fr) minmax(0, 2fr) minmax(0, 2fr)  minmax(0,1fr)">
+					<Box
+						ref={input1Ref}
+						type="text"
+						editable={greyOutPlacePoint()}
+						placeholder={greyOutPlacePoint() ? "Lat":"Disabled"}
+						content={greyOutPlacePoint() ? lat:""}
+						onChange={(e)=>setLat(e)}
+						style={{"backgroundColor":"#E2DBD5","borderColor":"transparent" }}
```

Comment:
It's probably a good idea to write a separate component for these boxes, since they're the same thing copy pasted multiple times. I would recommend that. You can put the component in the same file, somewhere below the main component (FlightPlanToolbar).